containers/Ideas/Idea.js

```diff
@@ -320,76 +320,78 @@ export const Idea = (props) => {
         onScroll={handleScroll}
         contentContainerStyle={styles.contentContainer}
       >
-        <View style={styles.titleContainer}>
-          <TextSourceSans style={styles.title}>{ideaState.name}</TextSourceSans>
-        </View>
-        <View style={styles.descriptionContainer}>
-          {ideaState.image && (
-            <>
-              <Image source={{ uri: ideaState.image }} style={styles.ideaImage} />
-            </>
-          )}
-          <TextSourceSans style={styles.text}>{ideaState.description}</TextSourceSans>
-        </View>
-        {getLabels().length > 0 && (
-          <View style={styles.labelsContainer}>
-            {getLabels().map((label, idx) => (
-              <Label key={idx + label} title={label} />
-            ))}
+        <Pressable onPress={isEditing && toggleEditing} style={{
+          ...isEditing ? {opacity: 0.25, backgroundColor: '#fff'}: {}
```

Comment:
maybe we could use the same lightbox overlay (greyish) which we have when opening the three dots menu?

Reply:
oh, yes, didn't see that! will check

Reply:
hmm, we are using a bottomsheet for that, I'm not sure how to integrate that nicely (it's also more grey than in the design I think?).

Reply:
i see, maybe at least we can change it to the design grey, instead of white? And see if we can adjust the grey of the bottomsheet?

Reply:
don't worry :) also if you know a good way to solve it with the bottomsheet feel free to push it here :)